src/utils/analytics/use-analytics.hook.ts

```diff
@@ -12,39 +12,45 @@ export const useAnalytics = () => {
   const userId = useUserIdSelector();
   const analyticsEnabled = useAnalyticsEnabledSelector();
 
-  const trackEvent = async (
-    event?: string,
-    category: AnalyticsEventCategory = AnalyticsEventCategory.General,
-    additionalProperties: AnalyticsEventProperties = {}
-  ) =>
-    event !== undefined &&
-    analyticsEnabled &&
-    jitsu.track(category, {
-      userId,
-      event,
-      timestamp: new Date().getTime(),
-      properties: {
+  const trackEvent = useCallback(
+    async (
+      event?: string,
+      category: AnalyticsEventCategory = AnalyticsEventCategory.General,
+      additionalProperties: AnalyticsEventProperties = {}
+    ) =>
+      event !== undefined &&
+      analyticsEnabled &&
+      jitsu.track(category, {
```

Comment:
Should the analytics change be in the scope of this PR? Please, let me know :)

Reply:
I think it should be there because it is a part of performance optimization